### ISSUE TITLE
How to get the valid set and test set about XNLI? 

### ISSUE THREAD

**Issue by 27182812** [ShannonAI/ChineseBert#23]
Hello，could you tell me how to get the valid set and test set about XNLI? In the link, I only found train set.

**Comment by xiaoya-li** [comment]
Hi, thanks for asking. <br>
As mentioned in the XNLI [project](https://github.com/facebookresearch/XNLI), the [XNLI-1.0.zip](https://dl.fbaipublicfiles.com/XNLI/XNLI-1.0.zip) contains train, dev and test files. <br>

**Comment by jiaqianjing** [comment]
Oh, I not found train set. Could you give me address about it？

**Comment by zzh-z** [comment]
hello, I'm trying to reproduce the XNLI experiment, could you please tell me how to get the Chinese training dataset?
Thanks!

**Comment by zzh-z** [comment]
hello, I'm trying to reproduce the XNLI experiment, could you please tell me how to get the Chinese training dataset?
Thanks!
